eyeshade/workers/reports.js

```diff
@@ -966,7 +966,12 @@ const prepareReferralPayout = async (debug, runtime, authority, reportId, thresh
 
     payment.owner = eligPublishers[i].owner
 
-    const entry = eligPublishers[i].ownerdata
+    const entries = eligPublishers[i].ownerdata
+    if (entries.length > 1) {
+      await notification(debug, runtime, payment.owner, payment.publisher, { type: 'verified_bad_ownerlink' })
```

Comment:
notification also notifies publishers, we should check what will happen if we send it a notification type it doesn't know about